lib/sha1.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+var inherits = require('inherits')
+var SHA = require('./sha')
+
+var ARRAY80 = new Array(80)
```

Comment:
Maybe just call this `var W = new Array(80)`,  and let it be super clear that it is a global used by `_getExpandedMessage`